working/templates/srmfunction/.template_config/template.json

```diff
@@ -41,7 +41,49 @@
       "parameters": {
         "format": "yyyy-MM-dd"
       }
-	}
+	},
+	"FunctionName": {
+        "type": "parameter",
+        "datatype": "string",
+		"defaultValue": "",
+        "replaces": "$FUNCTIONNAME$",
+        "description": "The name of the SRM function."
+    },
```

Comment:
Indeed, best to indicate the required items as required. Note: I think that will also mean that in VS you'll need to provide a default value (because VS doesn't show what is required if I recall correctly).